src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationQueryHandler.cs

```diff
@@ -32,9 +34,19 @@ public virtual async Task<PasswordValidationResponse> Handle(PasswordValidationQ
             foreach (var passwordValidator in userManager.PasswordValidators)
             {
                 var validationResult = await passwordValidator.ValidateAsync(userManager, null, request.Password);
-
                 result.Succeeded &= validationResult.Succeeded;
-                errorCodes.AddRange(validationResult.Errors.Select(x => x.Code));
+
+                result.Errors.AddRange(validationResult.Errors.Select(x =>
```

Comment:
Move function from the selector function to the protected virtual method.

Reply:
Why?